src/overrides/gemini-scrollbar.less

```diff
@@ -1,4 +1,5 @@
-@import (inline) "../../node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
+// Make sure to import the following path before including this file
+// @import (inline) "node_modules/react-gemini-scrollbar/node_modules/gemini-scrollbar/gemini-scrollbar.css";
```

Comment:
Correct. I added some docs to the readme and the kitchensink.
This really is the only way to do it that allows us to not install react because react is a peerDependency for react-gemini-scrollbar.